Cargo.toml

```
@@ -39,13 +42,13 @@ xmllint = ["proj"]
 
 [dependencies]
 chrono = "0.4"
-csv = "1.0"
-derivative = "1"
+csv = "1.1"
+derivative = "2.1"
```

How about we put it like this?

```suggestion
csv = "1"
derivative = "2"
```